<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MeasurementSavingDialog</class>
 <widget class="QDialog" name="MeasurementSavingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>592</width>
    <height>536</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Measurement form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="nameOfMeasurementLabel">
         <property name="text">
          <string>Name of measurement : </string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="nameOfMeasurementLineEdit"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="typeOfMeasurementLabel">
         <property name="text">
          <string>Type of measurement :</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="typeOfMeasurEdit">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="categoryLabel">
         <property name="text">
          <string>Category : </string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="categoryComboBox">
         <property name="editable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="temperatureLabel">
         <property name="text">
          <string>Temperature : </string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="temperatureLineEdit"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="commentsLabel">
         <property name="text">
          <string>Comments : </string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="textMeasResultLabel">
         <property name="text">
          <string>Measurement result :</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="measurResultLabel">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QTextEdit" name="commentsText"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="locale">
        <locale language="English" country="UnitedKingdom"/>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
